modules/apiserversource-yaml.adoc

```diff
@@ -14,10 +14,10 @@ This guide describes the steps required to create, manage, and delete an ApiServ
 
 .Procedure
 
-. Create a service account, role, and role binding for the ApiServerSource.
-+
-You can do this by creating a file named `authentication.yaml` and copying the following sample code into it:
+. Create a service account, role, and role binding for the ApiServerSource by creating a file named `authentication.yaml` and copying the following sample code into it:
```

Comment:
Nitpick, but it might read better with more active voice, such as:

> To create a service account, role, and role binding for the ApiServerSource,  create a file named `authentication.yaml` and copy the following sample code into it: